src/main/java/org/wiremock/grpc/dsl/WireMockGrpc.java

```diff
@@ -50,6 +52,17 @@ public static GrpcResponseDefinitionBuilder message(MessageOrBuilder messageOrBu
     return new GrpcResponseDefinitionBuilder(Status.OK).fromJson(json);
   }
 
+  public static GrpcResponseDefinitionBuilder messages(
+      List<MessageOrBuilder> messageOrBuilderList) {
+    final String json =
+        "[\n"
+            + messageOrBuilderList.stream()
```

Comment:
Is serialising the whole list not possible?

Seems a bit messy to have to do string concat like this.